Shokofin/Configuration/configPage.html

```diff
@@ -92,7 +92,7 @@ <h3>Metadata Settings</h3>
                         <legend>
                             <h3>Library Settings</h3>
                         </legend>
-                        <div class="fieldDescription verticalSection-extrabottompadding">"Automatically merge series that are spread across multiple folders" must be disabled in the <a href="#!/library.html">Library Settings</a> if the Show/Season grouping set to "Do not group Series into Seasons", otherwise it must be enabled. If you do not do this then the plugin will not work as expected.</div>
+                        <div class="fieldDescription verticalSection-extrabottompadding">If you wish to set the Series/Grouping setting below to "Group series into Season based on Shoko's Groups", you must disable "Automatically merge series that are stored in separate folders" in the setting for all Libraries that depend on ShokoFin for their metadata.<br>On the other hand, if you do not want to have Series and Grouping be determined by Shoko, then you must enable the "Automatically merge series that are stored in separate folders" setting for all Libraries that use ShokoFin for their metadata.<br>See the library settings under each individual Library here:<a href="#!/library.html">Library Settings</a></div>
```

Comment:
The option name to disable/enable is taken word-for-word from the library settings
![image](https://user-images.githubusercontent.com/7761729/166924108-f56f201d-acf1-4d57-8571-15d5e3b74643.png)

Reply:
no uppercase on the "Fin" in Shokofin

Reply:
Thank you - I found another error as well, and will address your comments as part of those changes.